moment/moment.d.ts

```diff
@@ -318,19 +318,27 @@ declare module moment {
 
     }
 
-    interface MomentLanguage {
+    interface BaseMomentLanguage { 
+        months ?: any;
```

Comment:
for consistency, I think the whitespace preceding all `?` characters should be removed